recipe/meta.yaml

```
@@ -10,50 +10,129 @@ source:
   sha256: 0715d4a0393437aa2a48c144756360f9b5292635a05fd15c38cbbaf04ef5acb9
 
 build:
-  entry_points:
-    - connexion = connexion.cli:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
 
-requirements:
-  host:
-    - python {{ python_min }}
-    - poetry-core >=1.2.0
-    - pip
+outputs:
+  - name: connexion-base
+    build:
+      entry_points:
+        - connexion = connexion.cli:main
+      noarch: python
+      script: recipe/install_connexion.sh
```

```suggestion
      script: install_connexion.sh
```
You don't need (and I believe can't have `recipe/`).

---

I had tried  `script: install_connexion.sh` but it reported that the file was not found.
That was why I tried it with `recipe/`.